composables/use-subscription.ts

```diff
@@ -130,6 +168,9 @@ export function useSubscription() {
         utmMedium: utmMedium || analyticsParams.utmMedium,
         utmSource: utmSource || analyticsParams.utmSource,
       })
+      if (redirectInfo && (redirectInfo.name || redirectInfo.path)) {
+        useStorage('plus_redirect_route', redirectInfo)
```

Comment:
I think `useStorage()` should put in `useAccountStore()`
and call `accountStore.savePlusRedirectRoute()`